doc/source/release-notes/Percona-Server-5.7.21-21.rst

```diff
@@ -0,0 +1,137 @@
+  Bug fixed :psbug:`1095`.
+
+* A memory leak took place in |Percona Server| when performance schema is used
+  in conjunction with thread pooling. Bug fixed :psbug:`1096`.
+
+* A code clean-up was done to fix compilation with clang, as general warnings 
+  (bug fixed :psbug:`3814`, upstream :mysqlbug:`89646`) so clang 6 specific
+  warnings and errors (bug fixed :psbug:`3893`, upstream :mysqlbug:`98111`).
+
+* Compilation warning was fixed for statically linked QRT. Bug fixed
+  :psbug:`3841`.
+
+* |Percona Server| returned empty result for ``SELECT`` query if number of
+  connections exceeded 65535. Bug fixed :psbug:`314`.
+
+* A clean-up in |Percona Server| binlog-related code was made to avoid
+  uninitialized memory comparison. Bug fixed :psbug:`3925` (upstream
+  :mysqlbug:`90238`).
+
+* ``mysqldump`` utility with ``--innodb-optimize-keys`` option was incorrectly
+  working with foreign keys on the same table, producing invalid SQL
+  statements. Bugs fixed :psbug:`1125` and :psbug:`3863`.
+
+* A fix of the mysqld startup script failed to detect jemalloc library
+  location for preloading, thus not starting on systemd based machines,
+  introduced in |Percona Server| :rn:`5.7.21-20`, was improved to take into
+  account previously created configuration file. Bug fixed :psbug:`3850`.
+
+* A fix introduced in |Percona Server| :rn:`5.7.10-2` which made busy server
+  not to prefer LRU flushing over flush list flushing too strongly was reverted
+  back because of the MT LRU flusher introduction, which removed LRU flushing
+  from the cleaner loop, Bug fixed :psbug:`3812`.
+
+* The possibility of a truncated bitmap file name was fixed in |InnoDB| logging
+  subsystem. Bug fixed :psbug:`3926`.
+
+* Non-instrumented functions were used in cached reads and writes of files in
+  fixed-size units. Bug fixed :psbug:`3937` (upstream :mysqlbug:`90264`).
+
+* A crash in the unsafe query warning checks with views took place for
+  ``UPDATE`` statement in case of statement binlogging format. Bug fixed
+  :psbug:`290`.
+
+MyRocks Changes
+===============
+
+* A re-implemented variable :variable:`rpl_skip_tx_api` allows to turn on simple
+  |RocksDB| write batches functionality, increasing replication performance
+  by the transaction api skip. Bug fixed :jirabug:`MYR-47`.
+
+* Unpack info bytes indicating the trailing space length in a padded varchar
+  were not read/skipped for value-less fields, causing data corruption and/or
+  server crash. Bug fixed :jirabug:`MYR-232`.
+
+TokuDB Changes
+===============
+
+* Two new variables introduced for the |TokuDB| fast updates feature,
+  :variable:`tokudb_enable_fast_update` and
+  :variable:`tokudb_enable_fast_upsert` should be now used instead of the 
+  ``NOAR`` keyword, which is now optional at compile time and off by default.
+  Bugs fixed :tdbbug:`63` and :tdbbug:`148`.
+
+* A set of compilation fixes as introduced to make |TokuDB| successfully
```

Comment:
8.0 -> MySQL / Percona Server 8.0

Reply:
I wonder if this is even worth noting here. There is (should be) no functional or user visible change as a result.

Reply:
I'd say this list of issues is here to show that a lot of work was done to make TokuDB compatible.